### DISCUSSION TITLE
web下载excel，打开后内容为空

### DISCUSSION THREAD

**Discussion by skyHu666** [alibaba/easyexcel#3528] [General]
response.setContentType("application/vnd.ms-excel");
        response.setCharacterEncoding("utf8");
        //2、设定附件的打开方法为：下载，并指定文件名称为category.xlsx  exportDataList
       response.setHeader("Content-disposition","attachment;filename="+ excelFileName +".xlsx");
        try(ServletOutputStream outputStream = response.getOutputStream()) {
            // 导出数据
            EasyExcel.write(outputStream).head(head(headList)).sheet(sheetName).doWrite(dataList);
        } catch (IOException e) {
            throw new RuntimeException("导出excel数据时异常！！！");
        }

**Comment by skyHu666** [comment]
[](url)
![企业微信截图_16985723042291](https://github.com/alibaba/easyexcel/assets/67506133/5496f5b3-a63c-4c8c-9a23-b1dc55a88b18)

**Reply by LSL1618** [reply]
`application/vnd.ms-excel`与`xlsx`完全对不上嘛！
